scripts/biomass_allometry_db.py

```diff
@@ -15,9 +18,10 @@ def __init__(self, **kwargs):
         self.ref = "https://doi.org/10.6084/m9.figshare.c.3307692.v1"
         self.urls = {"BAAD": "https://ndownloader.figshare.com/files/5634309"}
         self.citation = "Falster, D.S., Duursma, R.A., Ishihara, M.I., Barneche, D.R., FitzJohn, R.G., Varhammar, A., Aiba, M., Ando, M., Anten, N., Aspinwall, M.J. and Baltzer, J.L., 2015. BAAD: a Biomass And Allometry Database for woody plants."
+        self.licenses = [{"name": "CC0-1.0"}]
         self.keywords = ['plants', 'observational']
         self.retriever_minimum_version = "2.0.dev"
-        self.version = "1.4.0"
+        self.version = "1.4.2"
```

Comment:
version jump

Reply:
These scripts scripts/biomass_allometry_db.py, scripts/bird_size.json, scripts/amniote_life_hist.py have changes other than the python interface imports.

Reply:
OK. No harm in having them go in this way.